arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -0,0 +1,974 @@
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "namespaceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The namespace name."
+          },
+          {
+            "name": "authorizationRuleName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The connection string of the namespace for the specified authorizationRule."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RegenerateKeysParameters"
+            },
+            "description": "Parameters supplied to regenerate Auth Rule."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful",
+            "schema": {
+              "$ref": "#/definitions/ResourceListKeys"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CheckAvailabilityParameters": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Gets or sets name"
+        },
+        "location": {
+          "type": "string",
+          "description": "Gets or sets location."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets tags."
+        },
+        "isAvailiable": {
+          "type": "boolean",
+          "description": "Gets or sets true if the name is available and can be used to create new Namespace/ServiceBus. Otherwise false."
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "description": "Parameters supplied to the Check Name Availability for Namespace and ServiceBus."
+    },
+    "CheckAvailabilityResource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Gets or sets the id"
+        },
+        "location": {
+          "type": "string",
+          "description": "Gets or sets datacenter location"
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets or sets name"
+        },
+        "type": {
+          "type": "string",
+          "description": "Gets or sets resource type"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets tags"
+        },
+        "isAvailiable": {
```

Comment:
This is received in the response from the server. Do you think this should be settable v/s readonly?